partiql-ast/src/main/kotlin/org/partiql/ast/sql/internal/InternalSqlDialect.kt

```diff
@@ -82,6 +82,13 @@ internal abstract class InternalSqlDialect : AstBaseVisitor<InternalSqlBlock, In
             t = t concat ")"
             t
         }
+        is Expr.BagOp -> {
```

Comment:
Parens were missing from output to indicate precedence of the arguments.

E.g. previously an ast generated from the following query:
```
SELECT * FROM foo UNION (SELECT * FROM bar UNION SELECT * FROM baz)
```
would get printed as
```
SELECT * FROM foo UNION SELECT * FROM bar UNION SELECT * FROM baz
```